StatBot/EchoBot.cs

```diff
@@ -12,14 +12,14 @@
 using Microsoft.Bot.Builder.Dialogs;
 using Microsoft.Bot.Builder.Core.Extensions;
 using StatBot.Database.PostgresRepositories;
-
+using StatBot.Cards;
 
 namespace StatBot
 {
     public class EchoBot : IBot
     {
         private readonly DialogSet dialogs;
-
+        public static List<ActionButton> ShowedButtons = new List<ActionButton>();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'ShowedButtons' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2223)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use an immutable collection or reduce the accessibility of this field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2386)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this field 'private' and encapsulate it in a 'public' property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1104)